docs/changes/2418.bugfix.rst

```diff
@@ -1,2 +1 @@
-Do not use the hidden attribute `SKLearnReconstructor._models` in
-`ctapipe-apply-models`.
+Do not use a hidden attribute of `SKLearnReconstructor` in `ctapipe-apply-models`.
```

Comment:
You need double backticks here, single back ticks are references for sphinx, which we don't want in changelogs, since they might get invalidated in future updates.